### PR TITLE
docs: add doc describing collection schema

### PR DESCRIPTION
**Pull request by romanrostislavovich**
added doc describing collection schema

Part of #70

### CODE REVIEW

File: docs/schema/collection.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+|Key|Type|Required|Description|
+|-|:-:|:-:|-|
+| enum | string[] | + | An Array names of language |
+| extensions | string[] | - | An Array of associated extensions for that language |
+
+## Example
+
+```json
+{
+    "definitions":{
```
Comment (xferra) [inline docs/schema/collection.md:26]: This is not an example, but just a part of the schema.

File: docs/schema/collection.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+|-|:-:|:-:|-|
+| language | [language](#language)[] | - | Programming languages and associated extensions |
+| license | string[] | - | Software licenses names and acronyms |
+| manager | string[] | - | System and language package managers |
+
+### Language
+
+|Key|Type|Required|Description|
+|-|:-:|:-:|-|
+| enum | string[] | + | An Array names of language |
```
Comment (xferra) [inline docs/schema/collection.md:19]: Is language array names + array of extensions?

**Review by xferra — CHANGES_REQUESTED** [review]
I actually do not sure WHY this PR was created, coz there is no direct use of those objects and example do not demonstrate the structure or possible values. Let's keep it open while all other schemas will be finished. I recommend to document only core schemas and start with `package`.

**Comment by xferra** [comment]
@svoboda-rabstvo How do you think, do we still need this kind of doc?

**Comment by romanrostislavovich** [comment]
I think, no

**Comment by xferra** [comment]
Then please close PR with some descriptive comment 👍  (care: please do not close the related issue)

**Comment by xferra** [comment]
Closing this as far as docs for core schemas will provided in scope of another PR next week.

**Pull request closed without merging**